<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="translate_engine_label">
     <property name="text">
      <string>Translate Engine:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="translate_engine_combobox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="target_lang_label_1">
     <property name="text">
      <string>Target Language 1:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="translate_lang_combobox_1"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="target_lang_label_2">
     <property name="text">
      <string>Target Language 2:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="translate_lang_combobox_2"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="target_lang_label_3">
     <property name="text">
      <string>Target Language 3:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="translate_lang_combobox_3"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="font_label">
     <property name="text">
      <string>Font:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KFontRequester" name="font_kfontrequester">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
